cucumber.js

```diff
@@ -1,13 +1,43 @@
-var common = [
+const feature = [
   '--require-module ts-node/register',
   '--require features/**/*.ts',
   `--format ${
     process.env.CI || !process.stdout.isTTY ? 'progress' : 'progress-bar'
   }`,
   '--format rerun:@rerun.txt',
   '--format usage:usage.txt',
+  '--format message:messages.ndjson',
+].join(' ')
+
+const cck = [
+  '--require-module',
+  'ts-node/register',
+  '--format',
+  'message',
+].join(' ')
+
+const FORMATTERS_INCLUDE = [
+  'attachments',
+  'data-tables',
+  'minimal',
+  'parameter-types',
+  'rules',
+]
```

Comment:
How was this list formed?

Reply:
The ones that don't have conflicting step definitions, and don't cause everything else to fail. So everything except `hook` and `undefined-parameter-types` (realised I had missed two others, pushed a fix).